projects/go-lib/src/lib/components/go-datepicker/locale-format.spec.ts

```diff
@@ -13,5 +13,16 @@ describe('LocaleFormat', () => {
         dec.toUTCString() + ' is not a valid date'
       );
     });
+    it('should return false for a non leap year date', () => {
+      const nonLeapYear: Date = new Date(2021, 1, 29);
```

Comment:
I do not think we can do this this way as new Date(2021, 1, 29) will return a valid date this returns March 1st
So the test with be true